src/dune_lang/dune_lang.mli

```diff
@@ -105,6 +105,44 @@ end with type sexp := t
 
 val add_loc : t -> loc:Loc.t -> Ast.t
 
+(** Concrete syntax tree *)
+module Cst : sig
```

Comment:
`module Concrete` would make this a bit more obvious maybe?

Reply:
While the term CST is not used as often as AST, I believe it's common enough to justify keeping it here.